### PR TITLE
PMM-3732

### PR DESCRIPTION
**Pull request by Nailya**

### CODE REVIEW

File: pmm-tests/pmm-framework.sh [hunk 1]
```diff
@@ -706,6 +730,25 @@ setup(){
...
+  tar -xzf $PMM_CLIENT_TAR
+  PMM_CLIENT_BASEDIR=$(ls -1td pmm2-client-* 2>/dev/null | grep -v ".tar" | head -n1)
+  pushd $PMM_CLIENT_BASEDIR > /dev/null
+  export PATH="$PWD/bin:$PATH"
+
+}
+
+configure_client() {
+  agent_id=$(pmm-admin register --server-url="http://"$IP_ADDRESS:$PMM_PORT  --server-insecure-tls|sed -n 's/.*agent_id//p') 
+ sudo pmm-agent  --insecure-tls --id="/agent_id$agent_id" --address=$IP_ADDRESS:443 > $PWD/pmm-agent.logs &
```
Comment (puneet0191) [inline pmm-tests/pmm-framework.sh]: `sudo` needed here?

File: pmm-tests/pmm-framework.sh [hunk 2]
```diff
@@ -706,6 +730,25 @@ setup(){
...
+install_client(){
+ PMM_CLIENT_TAR_URL=$(lynx --listonly --dump https://www.percona.com/downloads/TESTING/pmm/ | grep  "pmm2-client" |awk '{print $2}'| grep "tar.gz" | head -n1)
+  echo "PMM2 client  $PMM_CLIENT_TAR_URL"
+  wget $PMM_CLIENT_TAR_URL
+  PMM_CLIENT_TAR=$(echo $PMM_CLIENT_TAR_URL | grep -o '[^/]*$')
+  tar -xzf $PMM_CLIENT_TAR
+  PMM_CLIENT_BASEDIR=$(ls -1td pmm2-client-* 2>/dev/null | grep -v ".tar" | head -n1)
+  pushd $PMM_CLIENT_BASEDIR > /dev/null
+  export PATH="$PWD/bin:$PATH"
+
```
Comment (puneet0191) [inline pmm-tests/pmm-framework.sh]: please fix indentation here and extra space

File: pmm-tests/pmm-framework.sh (hunk 1 above)
Comment (Nailya) [reply pmm-tests/pmm-framework.sh]: yes, it's known problem, going to be fixed

File: pmm-tests/pmm-framework.sh (hunk 2 above)
Comment (Nailya) [reply pmm-tests/pmm-framework.sh]: extra space and indentation were fixed

**Pull request merged**